tests/README.rst

```diff
@@ -1,106 +0,0 @@
-.. title: The Nikola Test Suite
-.. slug: tests
-.. date: 2012/03/30 23:00
-
-The Nikola Test Suite
```

Comment:
Is there a reason why you removed the tests README.rst completely?

Reply:
Ok, I see: "Remove terribly outdated test documentation"
Makes sense :)

Reply:
The document is fairly outdated, and after removing the now-irrelevant parts, it wouldn’t really be useful for anything.